package.json

```diff
@@ -26,7 +26,7 @@
   },
   "main": "./out/src/main",
   "activationEvents": [
-    "onDebug",
+    "onDebugResolve:type",
```

Comment:
That's all it takes?  Cool.

Reply:
I looked through the docs, changed this and debugging still worked so this looks right.

Reply:
Oh I need to update the engine version